nearai/solvers/__init__.py

```diff
@@ -79,24 +79,28 @@ def add_system_message(self, message: str) -> None:
         self.messages.append({"role": "system", "content": message})
 
     def run_task(self, task: str) -> str:
-        if self.agent:
-            assert self.env_run
-            self.env_run.run(task)
-            message = self.env_run.env.get_last_message(role="assistant")
-            return message.get("content") if message else ""
-        else:
-            self.messages.append({"role": "user", "content": task})
-            completion_response = cast(
-                ModelResponse,
-                self.client.completions(
-                    model=self.model_full_path,
-                    messages=self.messages,
-                    temperature=0.0,
-                ),
-            )
-            response_content = str(cast(List[Choices], completion_response.choices)[0].message.content)
-            self.messages.append({"role": "assistant", "content": response_content})
-            return response_content
+        try:
+            if self.agent:
+                assert self.env_run
+                self.env_run.run(task)
+                message = self.env_run.env.get_last_message(role="assistant")
+                return message.get("content") if message else ""
+            else:
+                self.messages.append({"role": "user", "content": task})
+                completion_response = cast(
+                    ModelResponse,
+                    self.client.completions(
+                        model=self.model_full_path,
+                        messages=self.messages,
+                        temperature=0.0,
+                    ),
+                )
+                response_content = str(cast(List[Choices], completion_response.choices)[0].message.content)
+                self.messages.append({"role": "assistant", "content": response_content})
+                return response_content
+        except Exception as e:
+            print(f"Error: {e}")
+            return " "
```

Comment:
We should return some standard error string.

Reply:
Solvers don't expect errors to be in model/agent output. " " is the most stable choice in case of inference errors for solvers.

Reply:
But it will not be logged this way.

Reply:
Changed to `f"{e}"`